geomstats/learning/riemannian_mean_shift.py

```diff
@@ -193,3 +193,21 @@ def predict(self, points):
             closest_centers.append(self.centers[closest_center, :])
 
         return gs.array(closest_centers)
+
+    def predict_labels(self, points):
+        """Predict the closest cluster label each point in 'points' belongs to.
+
+        Parameters
+        ----------
+        points : array-like, shape=[..., n_features]
+            Clusters of points.
+        """
+        if self.centers is None:
+            raise Exception("Not fitted")
+
+        closest_centers = []
+        for point in points:
```

Comment:
Ideally this should be vectorized, but it will imply changes in `metric.closest_neighbor_index`.